Source/Core/VideoCommon/RenderState.h

```diff
@@ -22,11 +22,18 @@ union RasterizationState
 {
   void Generate(const BPMemory& bp, PrimitiveType primitive_type);
 
-  RasterizationState& operator=(const RasterizationState& rhs);
+  RasterizationState() = default;
+  RasterizationState(const RasterizationState&) = default;
+  RasterizationState& operator=(const RasterizationState& rhs)
```

Comment:
I wonder if it'd be worthwhile to add `noexcept`

Reply:
Dolphin is built with exceptions disabled, isn't it?  Wouldn't `noexcept` be meaningless as such?

Reply:
I'm being nitpicky here, it probably doesn't matter, I just thought I'd raise it as I was curious.  I've recently been getting into the habit of marking at least my constructors noexcept (assuming they are that way) just as a "good practice".

Stackoverflow seems to indicate that there's very minor gains for using `noexcept` and it only applies in very specific use cases.  Otherwise it's solely a worthwhile addition as a marker for the programmer (just like `const` ).

I'm fine leaving it as is, just thought it'd be a good debate :)